packages/terra-application-links/src/tabs/_TabMenu.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import Popup from 'terra-popup';
 import { matchPath } from 'react-router-dom';
 import * as KeyCode from 'keycode-js';
-import { FormattedMessage } from 'react-intl';
+import { injectIntl } from 'react-intl';
```

Comment:
Would it be worth making this change throughout instead of concatenating the chunks array passed down by FormattedMessage?

Reply:
I tried to mostly switch to `injectIntl`, the problem is that the wrapper that `injectIntl` creates is a functional component, so when I made that change in a few of the components it caused issues if the `ref` for the component was being used in a test or example. `react-intl` v2 has a different api for forwarding refs than later versions.